template-ui/src/components/Breadcrumb.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div
+    v-if="!isBreadcrumbDisabled"
```

Comment:
One nitpick: "isXDisabled" sounds like we're checking some configuration to me, and double negative is :(

Perhaps…

```suggestion
    v-if="hasAncestors"
```

Reply:
Good point, I'm carrying the old logic here.

Reply:
I realized that we can just check `v-if="node.ancestors"` directly, without introducing a new method.